apps/snitch_core/test/support/order_case.ex

```diff
@@ -0,0 +1,60 @@
+defmodule Snitch.OrderCase do
+  @moduledoc """
+  Test helpers to insert stock items and locations.
+  """
+
+  @line_item %{
+    quantity: nil,
+    unit_price: nil,
+    total: nil,
+    variant_id: nil,
+    order_id: nil,
+    inserted_at: Ecto.DateTime.utc(),
+    updated_at: Ecto.DateTime.utc()
+  }
+
+  @doc """
+  Returns a list of line_item `map`s after zipping `variants` and
+  `quanities`.
+
+  The price fields are not computed. This result is suitable for a
```

Comment:
please put a note that the number of variants should be same as quantity.

Reply:
It need not be, and that's desirable. The zipping will stop when one of the enumerables is exhausted. A handy way of controlling the number of line_items -- just make the quantity list small :smile: